cluster-autoscaler/core/scaledown/actuation/delete_in_batch.go

```diff
@@ -195,6 +195,9 @@ func CleanUpAndRecordFailedScaleDownEvent(ctx *context.AutoscalingContext, node
 		klog.Errorf("Scale-down: couldn't delete empty node, %v, status error: %v", errMsg, status.Err)
 		ctx.Recorder.Eventf(node, apiv1.EventTypeWarning, "ScaleDownFailed", "failed to delete empty node: %v", status.Err)
 	}
+	gpuConfig := ctx.CloudProvider.GetNodeGpuConfig(node)
+	metricResourceName, metricGpuType := gpu.GetGpuInfoForMetrics(gpuConfig, ctx.CloudProvider.GetAvailableGPUTypes(), node, nil)
+	metrics.RegisterFailedScaleUp(metrics.CloudProviderError, metricResourceName, metricGpuType)
```

Comment:
This function is called to clean up after a failed scale *down*. Why would we register a failed scale *up* here?

Reply:
cool , will change

Reply:
changed , thanks !!